src/seedu/addressbook/ui/MainWindow.java

```diff
@@ -84,6 +85,10 @@ public void displayResult(CommandResult result) {
         if(resultPersons.isPresent()) {
             display(resultPersons.get());
         }
+        final Optional<ArrayList<String>> resultCommands = result.getRelevantCommands();
+        if(resultCommands.isPresent()) {
```

Comment:
Ya can leave it like that for now, but the coding standard for this module is to have a spacing there.
Even things like "for (int i=0; i<n; i++)" is not good, must be "for (int i = 0; i < n; i++)"